src/critics/path_align_critic.cpp

```diff
@@ -27,7 +27,6 @@ void PathAlignCritic::initialize()
 
 void PathAlignCritic::score(CriticData & data)
 {
-  utils::setPathFurthestPointIfNotSet(data);
```

Comment:
I believe this is set below (data.furthest_reached_path_point = max_s;), not sure it should be here. Seems like wasted computation, right @artofnothingness ?